nextflow.config

```diff
@@ -273,23 +273,93 @@ dag {
+            affiliation: """Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany""",
+            email: '',
+            github: '@ggabernet',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Erik Garrison',
+            affiliation: """University of Tennessee Health Science Center, Memphis, Tennessee, TN, USA""",
+            email: '',
+            github: '@ekg',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Andrea Guarracino',
+            affiliation: """University of Tennessee Health Science Center, Memphis, Tennessee, TN, USA""",
+            email: '',
+            github: '@AndreaGuarracino',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Friederike Hanssen',
+            affiliation: """Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany""",
+            email: '',
+            github: '@FriederikeHanssen',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Peter Heringer',
+            affiliation: """Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany""",
+            email: '',
+            github: '@heringerp',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Michael L Heuer',
+            affiliation: """Mammoth Biosciences, Inc., San Francisco, CA, USA""",
+            email: '',
+            github: '@heuermh',
+            contribution: ['author', 'maintainer'],
+            orcid: 'https://orcid.org/0000-0002-9052-6000'
+        ],
+        [
+            name: 'Lukas Heumos',
+            affiliation: """Institute of Computational Biology, Helmholtz Zentrum München, Munich, Germany""",
+            email: '',
+            github: '@zethson',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
         [
             name: 'Simon Heumos',
-            affiliation: '',
+            affiliation: """Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany""",
+            email: '',
+            github: '@subwaystation',
+            contribution: ['author', 'maintainer'],
+            orcid: 'https://orcid.org/0000-0003-3326-817X'
+        ],
+        [
+            name: 'Susanne Jodoin',
+            affiliation: """Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany""",
             email: '',
-            github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            github: '@SusiJo',
+            contribution: ['contributor'],
             orcid: ''
         ],
         [
-            name: ' Michael L Heuer',
-            affiliation: '',
+            name: 'Júlia Mir Petrol',
```

Comment:
😉 
```suggestion
            name: 'Júlia Mir Pedrol',
```

Reply:
Thank you!  I copied from README, where it was also wrong!  Updating in next commit...